Github/service.js

```diff
@@ -11,6 +11,7 @@
 const r = require('request');
 
 const { key } = require('../config-files/github-token');
+const { repos } = require('../config-files/github-whitelist');
```

Comment:
I think the `repos` naming can be a bit less ambiguous to indicate that these are whitelisted repos. A couple options I'd suggest:

1) Rename during destructuring (i.e. `const { repos: whitelist } = require('../config-files/github-whitelist')`)

2) Or alternatively, change `whitelist.json` to:  `[ "repo1", "repo2". "repo3"]` and import with `const whitelist =  require('../config-files/github-whitelist.json')`